modules/hostap/src/supp_events.h

```diff
@@ -11,7 +11,7 @@
 
 /* Connectivity Events */
 #define NET_MGMT_SUPPLICANT_LAYER NET_MGMT_LAYER_L3
-#define NET_MGMT_SUPPLICANT_CODE 0x157
+#define NET_MGMT_SUPPLICANT_CODE NET_MGMT_LAYER_CODE_WIFI
```

Comment:
Noticed that I set the layer incorrectly for supplicant events, will fix that after CI has passed 1st time.